src/api-requests/get-all-search-content.ts

```diff
@@ -1,12 +1,46 @@
-import axios from 'axios';
+import {
+    buildSearchQuery,
+    SearchQueryParams,
+} from '../components/search/utils';
 import { SearchItem } from '../components/search/types';
 
 // Should find a way to cache this response. Only use it in a few places for search filters but still.
 export const getAllSearchContent = async (): Promise<SearchItem[]> => {
```

Comment:
`getAllSearchContent` and `getSearchContent` are two very similar functions. Could we possibly look into combining the two that would just take in that `queryParams` arg?

Reply:
Agreed, will fix that.